internal/pkg/scalable/cronjobs.go

```diff
@@ -21,11 +21,27 @@ func getCronJobs(namespace string, clientsets *Clientsets, ctx context.Context)
 	return results, nil
 }
 
+// getCronJob is the getResourceFunc for a single CronJob
+func getCronJob(name string, namespace string, clientsets *Clientsets, ctx context.Context) (Workload, error) {
+	var result Workload
+	cronjob, err := clientsets.Kubernetes.BatchV1().CronJobs(namespace).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return result, fmt.Errorf("failed to get cronjob: %w", err)
+	}
+	result = &suspendScaledWorkload{&cronJob{cronjob}}
+	return result, nil
+}
```

Comment:
I really think we shouldn't have a whole other function just to get just 1 of the resource.

```suggestion
// getCronJobs is the getResourceFunc for CronJobs
func getCronJobs(name, namespace string, clientsets *Clientsets, ctx context.Context) ([]Workload, error) {
    var results []Workload
    if name != "" {
        cronjob, err := clientsets.Kubernetes.BatchV1().CronJobs(namespace).Get(ctx, name, metav1.GetOptions{})
        if err != nil {
            return result, fmt.Errorf("failed to get cronjob: %w", err)
        }
        results = append(results, &suspendScaledWorkload{&cronJob{cronjob}})
        return results
    }

    cronjobs, err := clientsets.Kubernetes.BatchV1().CronJobs(namespace).List(ctx, metav1.ListOptions{TimeoutSeconds: &timeout})
    if err != nil {
        return nil, fmt.Errorf("failed to get cronjobs: %w", err)
    }
    for _, item := range cronjobs.Items {
        results = append(results, &suspendScaledWorkload{&cronJob{&item}})
    }
    return results, nil
}
```

I think we should keep the client.GetWorkload and GetWorkloads in single functions. This will also help abstract this configuration away.